ethereum/executionclient/src/main/java/tech/pegasys/teku/ethereum/executionclient/web3j/JsonRpcErrorCodes.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Consensys Software Inc., 2022
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.ethereum.executionclient.web3j;
+
+public enum JsonRpcErrorCodes {
+  PARSE_ERROR(-32700, "Parse error"),
+  INVALID_REQUEST(-32600, "Invalid Request"),
+  METHOD_NOT_FOUND(-32601, "Method not found"),
+  INVALID_PARAMS(-32602, "Invalid params"),
+  INTERNAL_ERROR(-32603, "Internal error"),
+  SERVER_ERROR(-32000, "Server error");
+
+  private final int errorCode;
+  private final String description;
+
+  JsonRpcErrorCodes(final int errorCode, final String description) {
+    this.errorCode = errorCode;
+    this.description = description;
+  }
+
+  public int getErrorCode() {
+    return errorCode;
+  }
+
+  public String getDescription() {
+    return description;
+  }
+
+  public static String getErrorMessage(final int errorCode) {
+    return fromCode(errorCode).getDescription();
+  }
+
+  public static JsonRpcErrorCodes fromCode(final int errorCode) {
+    for (JsonRpcErrorCodes error : values()) {
+      if (error.getErrorCode() == errorCode) {
+        return error;
+      }
```

Comment:
I suspect to make this better we could have a static `Int2ObjectOpenHashMap` map that could be fed in the `JsonRpcErrorCodes` constructor.

Reply:
Okay @tbenr 
Let me know if this is what we are aiming for 
![image](https://github.com/user-attachments/assets/eca042aa-3e31-415a-ab04-b7f95080c355)

Reply:
@Malaydewangan09  yes! My suggestion wasn't totally correct but you got it right

Reply:
Okay @tbenr 
Thanks!